service/build.gradle.kts

```diff
@@ -90,8 +89,6 @@ val shadowJar by tasks.getting(com.github.jengelman.gradle.plugins.shadow.tasks.
   exclude("module-info.class") // https://github.com/johnrengelman/shadow/issues/352
   // https://youtrack.jetbrains.com/issue/KT-25709
   exclude("**/*.kotlin_metadata")
-  exclude("**/*.kotlin_module")
-  exclude("**/*.kotlin_builtins")
```

Comment:
These exclusions caused the runtime to lose type information about kotlin stdlib types like `kotlin.Any`.

Reply:
Are you using the shaded jar? We stopped using it. Could remove this section if you're not. This exclusion had to be done to work around https://youtrack.jetbrains.com/issue/KT-25709

Reply:
Hm we usually use shaded jars. Removing these particular excludes allow the shaded jar to start up and doesn't appear to have any other negative impact. This section now pretty closely matches the latest comment on that youtrack issue.

Reply:
if it's working for you with this then :+1:, just wanted to give headsup